composer.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "ricbra/php-discogs-api",
+    "name": "davalb/php-discogs-api",
```

Comment:
Hi @ricbra, sorry about the second commit. I was hoping it wouldn't be attached to the PR. I will remove it now.
Yes, please tell me how to configure my composer.json, so I can install my fork. That is what I was trying to do with the package. Thanks!